### PR TITLE
gcp and azure tests fails fixed

### PR DESCRIPTION
**Pull request by ganeshjagdale9545**

### CODE REVIEW

File: cartography/data/indexes.cypher [hunk 1]
```diff
@@ -254,10 +254,10 @@ CREATE INDEX ON :AzureVirtualMachineAvailableSize(id);
 CREATE INDEX ON :AzureVirtualMachineScaleSet(id);
 CREATE INDEX ON :AzureVirtualMachineScaleSetExtension(id);
 CREATE INDEX ON :AzureKeyVault(id);
-CREATE INDEX ON :AzureUser(name);
```
Comment (mpurusottamc) [inline cartography/data/indexes.cypher:257]: did we change the field name from name to id?

File: cartography/data/indexes.cypher [hunk 2]
```diff
@@ -254,10 +254,10 @@ CREATE INDEX ON :AzureVirtualMachineAvailableSize(id);
...
+CREATE INDEX ON :AzureUser(id);
 CREATE INDEX ON :AzureGroup(id);
 CREATE INDEX ON :AzureApplication(id);
-CREATE INDEX ON :AzureServiceAccount(name);
+CREATE INDEX ON :AzureServiceAccount(id);
```
Comment (mpurusottamc) [inline cartography/data/indexes.cypher:260]: did we change the field name from name to id?

File: cartography/data/jobs/cleanup/azure_tenant_cleanup.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {
 	"statements": [
-	{
-		"query": "MATCH (n:AzureTenant{id: {AZURE_TENANT_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
-		"iterative": true,
-		"iterationsize": 100
-	}],
+		{
+			"query": "MATCH (n:AzureTenant) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```
Comment (mpurusottamc) [inline cartography/data/jobs/cleanup/azure_tenant_cleanup.json]: we should not remove this from the cleanup file. instead, we should pass this as part of the cleanup configuration object.

File: cartography/data/jobs/cleanup/gcp_api_configs_cleanup.json [hunk 4]
```diff
@@ -1,17 +1,17 @@
 {
     "statements": [
         {
-            "query": "MATCH (n:GCPAPIConfig)<-[:HAS_CONFIG]-(:GCPAPI)<-[:RESOURCE]-(:GCPProject{id: {GCP_PROJECT_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+            "query": "MATCH (n:GCPAPIConfig)<-[:HAS_CONFIG]-(:GCPAPI)<-[:RESOURCE]-(:GCPProject) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```
Comment (mpurusottamc) [inline cartography/data/jobs/cleanup/gcp_api_configs_cleanup.json]: we should not remove this parameter `GCP_PROJECT_ID` from the cleanup file. instead, we should pass this as part of the cleanup configuration object.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
Minor changes needed. Also, few other tests are failing.

File: cartography/data/indexes.cypher (hunk 1 above)
Comment (ganeshjagdale9545) [reply cartography/data/indexes.cypher:257]: yes

**Review by mpurusottamc — APPROVED** [review]
this is great. all the test failures are resolved now.

**Pull request merged**